guides/doc-Provisioning_Guide/topics/proc_configuring_iPXE_to_reduce_provisioning_times.adoc

```diff
@@ -0,0 +1,348 @@
+. Locate the *Default PXE global template entry* row and in the *Value* column, change the value to *discovery*.
+
+=== Example WIP of Lzap's Blogpost
+
+Using {Project} you can set up and use iPXE if you have the following restrictions that prevent you from using PXE:
+
+* On networks with unmanaged DHCP servers,
+* If the PXE service is blacklisted on your network or restricted by a firewall
+* When the TFTP UDP-based protocol is not reliable, for example, on low-bandwidth networks
+
+With {Project}, you can set up hosts to download either the `ipxe.efi` or `undionly.kpxe` over TFTP. When the file downloads, all communication continues using HTTP. {Project} uses the iPXE provisioning script either to load an operating system installer or the next entry in the boot order.
+
+You can use this workflow with or without discovery phase. If you want to use discovery, see xref:setting_up_the_discovery_service_for_iPXE[].
+
+.Security Information
+
+The iPXE binary in {RHEL} is built without some security features, for this reason, you can use HTTP, and cannot use HTTPs. All security-related features of iPXE in {RHEL} are not supported. For more information, see https://access.redhat.com/solutions/3483601[Red Hat Enterprise Linux HTTPS support in iPXE].
+
+.Prerequisite
+
+Ensure that the hardware that you use is compatible with iPXE.
+
+.Procedure
+
+. Copy the `ipxe.efi` file to the TFTP directory. For BIOS systems, copy the `undionly.kpxe` file.
++
+----
+# cp /usr/share/ipxe/undionly.kpxe /var/lib/tftpboot/undionly.0
+# cp /usr/share/ipxe/ipxe.efi /var/lib/tftpboot/
+----
++
+. In the {Project} web UI, navigate to *Hosts* > *Provisioning Templates*, enter `Kickstart default iPXE` and click *Search*.
+. Optional: If you want to change the template, click *Clone*, enter a unique name, and click *Submit*.
+. Click the name of the template you want to use.
+. If you clone the template, you can make changes you require on the *Template* tab.
+. Click the *Association* tab, and select the operating systems that your host uses.
+. Click the *Locations* tab, and add the location where the host resides.
+. Click the *Organizations* tab, and add the organization that the host belongs to.
+. Click *Submit* to save the changes.
+. Navigate to *Hosts* > *Operating systems* and select the operating system of your host.
+. Click the *Templates* tab.
+. From the *iPXE Template* list, select the template you want to use.
+. Click *Submit* to save the changes.
+. Navigate to *Hosts* > *All Hosts*.
+. In the *Hosts* page, select the host that you want to use.
+. Select the *Templates* tab.
+. From the *iPXE template* list, select *Review* and verify that the *Kickstart default iPXE* template is the correct template.
+. To use the iPXE bootstrapping feature for {Project}, configure the `dhcpd.conf` file as follows:
++
+----
+if exists user-class and option user-class = "iPXE" {
+  filename "http://foreman.nat.lan/unattended/iPXE?bootstrap=1";
```

Comment:
@lzap - should I edit this to read http://foreman.nat.lan/unattended/?bootstrap=1 ?